packages/cf-style-container/src/createComponent.js

```diff
@@ -0,0 +1,72 @@
+import { extractPassThroughProps, resolvePassThrough } from 'fela-utils';
+import { createElement } from 'react';
+import PropTypes from 'prop-types';
+
+function createComponentFactory(createElement, contextTypes) {
+  return function createComponent(rule, type, passThroughProps) {
+    const displayName = rule.name ? rule.name : 'FelaComponent';
```

Comment:
Would this be simpler as `const displayName = rule.name || 'FelaComponent'`?

Could we also add a comment about when `rule.name` would be falsy? I can imagine someone seeing a lot of `FelaComponent`s in their code and wanting to know why, digging through the code to find this line so it would be good to have a comment explaining why it's happening?